### ISSUE TITLE
Users search in online demo project

### ISSUE THREAD

**Issue by MMR100** [opengisch/QField#596]
When searching for user "Toto" in online demo project, the map is blocked (the point seems to be on the top left corner of the screen but I cannot move it anymore). 
To unblock the screen, I have to search for a valid user (e.g. "Test" is working). 
If I don't search for a new user, the screen remain blocked. 

I thought it was only my mistake but I realised the same issues occurs when searching for "Hhtv" user. 

**How to Reproduce**

1. Open online demo project showing Qfield users around the world 
2. Search for "Toto" or "Hhtv" 
3. The map cannot be moved
4. Search for another user
5. The map move again and zoom on the selected user.

I m using QFIELD 1.1.0 and a Samsung Galaxy J6

**Comment by signedav** [comment]
When I search for the user "Toto" and I click on it nothing happens. But I still can tip on the map and move around. There is a second valid user listed "TOTO - TOTOSP" on click on it I come to his position.
But this I cannot reproduce: "If I don't search for a new user, the screen remain blocked." I am able to move.

Could you test with the latest master version  [here](https://github.com/opengisch/QField/commit/3249a9ac1ee1e4fe38f549b6f58663cd538391d3#commitcomment-35774953) @MMR100 ?

**Comment by signedav** [comment]
> (the point seems to be on the top left corner of the screen but I cannot move it anymore)
This is possibly another issue...

**Comment by MMR100** [comment]
With latest master version, clicking on  "Toto" is showing a popup "l'entité ne possède pas de géométrie" (feature without geometry). My screen is not blocked anymore. The point is not on the top left corner of the screen anymore.

**Comment by MMR100** [comment]
After searching and clicking on "Toto", if I click on the square to zoom on feature, it is moving to the middle of Atlantic Ocean (near another user "Mani S - AED")

**Comment by signedav** [comment]
Yes. I can confirm the message about the geometry. But On clickign the "form"-button, the form opens without going anywhere. Strange. Keep us up to date if you have more information. I will do some research as well.

**Comment by MMR100** [comment]
![image](https://user-images.githubusercontent.com/26409106/68308682-935e6280-00ad-11ea-92f0-96a7373b19fc.png)
After clicking several time in the search bar on the feature without geometry, the form appears. Then I click on the square button circled in red in the attached picture. The screen is moved. I just checked with another point without geometry I just found in the search bar (I renamed it "test - no geometry), it is moving to the same location.
